packages/expo-live-photo/ios/ExpoLivePhoto.podspec

```diff
@@ -0,0 +1,29 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'ExpoLivePhoto'
+  s.version        = package['version']
+  s.summary        = package['description']
+  s.description    = package['description']
+  s.license        = package['license']
+  s.author         = package['author']
+  s.homepage       = package['homepage']
+  s.platforms      = {
+    :ios => '15.1',
+    :tvos => '15.1'
```

Comment:
Have you tested that on tvOS? 👀

Reply:
Nope, just copied and pasted the podspec from another package and missed this 😅

Reply:
Let's remove it then 😅